packages/backend/src/constants.ts

```diff
@@ -3,3 +3,37 @@ export const API_VERSION = 'v1';
 
 // TODO: Get information from package.json or define them from ESBuild
 export const USER_AGENT = `@clerk/backend`;
+
+const Attributes = {
+  AuthStatus: '__clerkAuthStatus',
+} as const;
+
+const Cookies = {
+  Session: '__session',
+  ClientUat: '__client_uat',
+} as const;
+
+const Headers = {
+  AuthStatus: 'x-clerk-auth-status',
+  AuthReason: 'x-clerk-auth-reasons',
+  AuthMessage: 'x-clerk-auth-message',
+  NextRewrite: 'x-middleware-rewrite',
```

Comment:
🔧 I am not a huge fan of adding framework-specific constants in the @clerk/backend package as this is the first minor stain that demonstrates framework knowledge in a framework-agnostic package.

Reply:
You're 100% correct, will move them back with the next commit